frontend/src/app/docs/api-docs/api-docs.component.ts

```diff
@@ -87,6 +85,14 @@ export class ApiDocsComponent implements OnInit, AfterViewInit {
     });
   }
 
+  ngOnDestroy(): void {
```

Comment:
Hm, we didn't import or implement `OnDestroy` from `angular/core` 🤔 , can we use `ngOnDestroy` without doing that? 🤔

Reply:
We can't, but this changed after merge and is not present in master.

Reply:
@softsimon Ah, thanks for the clarification.. didn't use angular for years so just wanted to confirm 👍  :)